tutorials/link-labels.html

```diff
@@ -289,8 +307,15 @@ <h3 id="label-rotation">Label Rotation</h3>
             <h3 id="label-styling">Label Styling</h3>
 
             <p>Of course, it is also possible to change the appearance of your labels.
-                Simply specify custom <code>markup</code> in JSON format and pass some <code>attrs</code>.
+                To specify <a href="/docs/jointjs#dia.Link.labels.markup" target="_blank">custom markup</a>, you may
+                provide a <code>markup</code> object in JSON format or as a <code>joint.util.svg</code>
+                <a href="/docs/jointjs#util.svg" target="_blank">ES6 tag template</a>.
                 As a bonus, you can define custom selectors to identify individual components of your label.
+                To specify <a href="/docs/jointjs#dia.Link.labels.styling" target="_blank">custom attributes</a>, you
+                may provide an <code>attrs</code> object with SVG attributes or
+                <a href="special-attributes.html">JointJS special attributes</a>.
+                Finally, to specify <a href="/docs/jointjs#dia.Link.labels.size" target="_blank">custom size</a> of the
```

Comment:
"Finally, to specify **the**..."

Reply:
I think this is actually fine, considering context:
- "To specify custom markup, you may..."
- "To specify custom attributes, you may..."
- "Finally, to specify custom size of the label, you may..."